src/deepsparse/pipeline.py

```diff
@@ -199,7 +199,7 @@ def create(
         task: str,
         model_path: str = None,
         engine_type: str = DEEPSPARSE_ENGINE,
-        batch_size: Optional[int] = None,
+        batch_size: int = 1,
```

Comment:
i thought we want to be dynamic by default - does this break something?

Reply:
resolved offline - net net pipelines are not required to be dynamic so avoiding making this default